rmrl/__main__.py

```diff
@@ -44,10 +52,18 @@ def main():
     stream = render(source,
                     template_alpha=float(args.alpha),
                     expand_pages=not args.no_expand,
-                    only_annotated=args.only_annotated)
+                    only_annotated=args.only_annotated,
+                    black=parse_color(args.black),
+                    white=parse_color(args.white))
```

Comment:
In anticipation of making the grey and highlighter colors configurable, I'm thinking we should pass a color dictionary through all of the layers.  (Or maybe a namedtuple?  Or dataclass?)

Reply:
Hm.  I think a namedtuple probably makes the most sense, with keys for black, white, grey, and highlight.